extensions/vscode/src/extension.ts

```diff
@@ -4,8 +4,12 @@ import * as vscode from "vscode";
  * This method is called when the extension is activated.
  *
  * The extension is activated the very first time the command is executed.
- * @param context
+ *
+ * @param {vscode.ExtensionContext} context
+ * @returns {vscode.ExtensionContext}
```

Comment:
Correct me if I am wrong, but these are no longer needed if you are using typescript right?

We should define a return type for this method on the method definition itself

Reply:
I've moved the type to the function declaration. Thanks for catching that! 

I've left both @param and @returns to comply with [TS Doc](https://tsdoc.org/). Including them does alter the output when hovering on the functions.